### PR TITLE
compute metrics using logits for loss and predictions for accuracy

### PR DESCRIPTION
**Pull request by Ranapop**

### CODE REVIEW

File: examples/cfq/input_pipeline_utils.py [hunk 1]
```diff
@@ -26,28 +26,29 @@
...
-
-  for dataset in datasets:
-    # Apply the tokenizer to all input features.
-    tokenized_dataset = dataset.map(_tokenize_input_features,
-                                    num_parallel_calls=constants.AUTOTUNE)
-    # Yield all tokenized input features (i.e., tokenized input sentences).
-    for example in tfds.as_numpy(tokenized_dataset):
-      for feature in input_features:
-        yield example[feature]
+    """Returns a list of tokens for all input fields in the given datasets."""
```
Comment (marcvanzee) [inline examples/cfq/input_pipeline_utils.py]: The indentation is wrong again in this file.

**Review by marcvanzee — APPROVED** [review]
Looks good! Please fix the indentation but otherwise LGTM.
